NuGet.config

```diff
@@ -26,7 +26,8 @@
     <add key="vs-editor" value="https://myget.org/F/vs-editor/api/v3/index.json" />
     <add key="roslyn_concord" value="https://myget.org/F/roslyn_concord/api/v3/index.json" />
     <add key="vssdk" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vssdk/nuget/v3/index.json" />
-    <add key="vs-impl" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json" />
+    <add key="vs-impl" value="https://devdiv.pkgs.visualstudio.com/_packaging/vs-impl/nuget/v3/index.json" />
+    <add key="vs-impl-public" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json" />
```

Comment:
><add key="vs-impl-public" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.jso [](start = 4, length = 116)

Why are there two feeds?

Reply:
@AArnott @JoshuaBStevens

Reply:
`https://devdiv.pkgs.visualstudio.com/_packaging/vs-impl/nuget/v3/index.json` should probably be removed as that is a private feed and I think roslyn wants to only consume public ones (which the latter one on `azure-public` is).